### ISSUE TITLE
[Firestore] context.eventType does not match documentation

### ISSUE THREAD

**Issue by frunkad** [firebase/firebase-functions#739]
### [REQUIRED] Step 2: Describe your environment

  * Operating System version: Linux Mint 19.1 Tessa
  * Firebase SDK version: 8.4.2
  * Firebase Product: Firestore
  * Node.js version: v12.11.1 (deployed to node 10)
  * NPM version: 6.11.3

### [REQUIRED] Step 3: Describe the problem

The `context.eventType` returned after a firestore background trigger run, should give one of the values described [here](https://firebase.google.com/docs/reference/functions/cloud_functions_.eventcontext#eventtype), or one among these:

```
providers/cloud.firestore/eventTypes/document.write
providers/cloud.firestore/eventTypes/document.create
providers/cloud.firestore/eventTypes/document.update
providers/cloud.firestore/eventTypes/document.delete
```

But on running, it returns the following values:

```
google.firestore.document.write
google.firestore.document.create
google.firestore.document.update
google.firestore.document.delete
```

#### Relevant Code:

```
import * as functions from 'firebase-functions';

export const documentWrite = functions.firestore.document('users/{uid}').onWrite((snap, ctx) => {
    console.log(ctx.eventType);
})
export const documentCreate = functions.firestore.document('users/{uid}').onCreate((snap, ctx) => {
    console.log(ctx.eventType);
})
export const documentUpdate = functions.firestore.document('users/{uid}').onUpdate((snap, ctx) => {
    console.log(ctx.eventType);
})
export const documentDelete = functions.firestore.document('users/{uid}').onDelete((snap, ctx) => {
    console.log(ctx.eventType);
})
```

Deployed via running `firebase deploy --only functions`

#### Log Screenshot

![Log Screenshot](https://user-images.githubusercontent.com/33373582/87528851-bbc24600-c6ab-11ea-840a-3e1669905972.png)

**Comment by schmidt-sebastian** [comment]
@laurenzlong Can you take a look?

**Comment by frunkad** [comment]
At 

https://github.com/firebase/firebase-functions/blob/df35c1b4b18b7f2cd6c2d0d21090793b3b153e8b/src/providers/firestore.ts#L250

It says these are legacy event types. Thus I believe it would require a comment update at [cloud-functions.ts](https://github.com/firebase/firebase-functions/blob/e447949d0d6095b130ca5551e3396b45ff5d589a/src/cloud-functions.ts) + maybe an update at [reference doc](https://firebase.google.com/docs/reference/functions/cloud_functions_.eventcontext#eventtype)

It being an easy issue + open source, can I take it up?

**Comment by laurenzlong** [comment]
Thanks for bringing these up! These are the correct event types values within `EventContext`:
```
google.firestore.document.write
google.firestore.document.create
google.firestore.document.update
google.firestore.document.delete
```
This is indeed a documentation issue, I wrote a fix in https://github.com/firebase/firebase-functions/pull/743.

**Comment by inlined** [comment]
It looks like this bug was fixed in #743 but never closed.
